api/src/modules/proxy/proxy.service.ts

```diff
@@ -9,6 +9,9 @@ import * as Server from 'http-proxy';
 
 @Injectable()
 export class ProxyService {
+  private geoprocessingServiceUrl: string = AppConfig.get(
+    'geoprocessing.url',
+  ) as string;
```

Comment:
Similar to logger stuff (injecting). Current approach makes it really hard to test it in an unit way. 

([one of the example reading, you can find plenty of those](https://ducmanhphan.github.io/2020-01-15-Understanding-about-SOLID-part-5/))